src/theory/arith/theory_arith_private.h

```diff
@@ -316,6 +316,8 @@ class TheoryArithPrivate {
   /** For holding the proof of the above conflict node. */
   context::CDO<std::shared_ptr<ProofNode>> d_blackBoxConflictPf;
 
+  bool isProofEnabled() const { return d_pnm != nullptr; }
```

Comment:
Can you move the implementation to the cpp?

Reply:
Sure thing.